packages/conversation/tests/rest/v1/conversation/conversation-api.test.ts

```diff
@@ -10,6 +10,10 @@ import {
   ListConversationsRequestData,
   StopActiveConversationRequestData,
   UpdateConversationRequestData,
+  ListRecentConversationsRequestData,
```

Comment:
What are the differences between  `*RequestData` and `*Request` type usage ?

Reply:
`*RequestData` are the interfaces for the request... data 😁 , meaning the grouping of the path, query and body parameters.
`*Request` would be an interface for a body object